### ISSUE TITLE
Could not resolve required module ’esm’

### ISSUE THREAD

**Issue by bernd-wechner** [kamataryo/fetch-to-curl#2]
Thanks for this. I'm keen to give it a test spin. 

Alas the error in the subject befuddles me. I did:

```
$ git clone https://github.com/kamataryo/fetch-to-curl.git
$ cd fetch-to-curl
$ npm test
```

then:

```
$ sudo npm install -g ava
$ sudo npm install -g esm
```
But still I get `Could not resolve required module ’esm’` when I run `npm test`

A noob to node so apologies if it's a simply issue but appreciate a pointer of course.

**Comment by kamataryo** [comment]
@bernd-wechner Hi, thanks for reporting!
It seems to be caused by the documentation lacking. We need to run `npm install` before executing the `npm test`. I have just fixed the documentation.

https://github.com/kamataryo/fetch-to-curl/commit/a01de3602578e92ca10622613a9b4133f37e334d

The npm dependencies of this project are old now, but I found that `npm test` successes after install with my environment.

As additional information, I recently prefer to use a browser option to get the curl command for debugging, which appears at Network -> each item -> (right-click) -> Copy -> Copy as cURL.

<img width="800" alt="Screenshot 2020-06-03 10 52 32" src="https://user-images.githubusercontent.com/6292312/83586864-8d1f5e80-a588-11ea-84be-ba5559a47fd5.png">

Feel free to open the issue again if you have any issues!
